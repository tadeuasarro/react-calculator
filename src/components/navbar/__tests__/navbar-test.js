import { BrowserRouter } from 'react-router-dom';
import { render, screen } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Navbar from '../navbar';

describe('Quote', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('creates a Quote component', () => {
    render(<BrowserRouter><Navbar /></BrowserRouter>);
    const element = screen.getByText('Math-magicians');
    expect(element).toBeInTheDocument();
  });
});
