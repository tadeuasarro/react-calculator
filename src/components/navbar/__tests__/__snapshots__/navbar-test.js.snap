// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders correctly 1`] = `
<div
  id="navbar"
>
  <h1>
    Math-magicians
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/app"
        onClick={[Function]}
      >
        App
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
